inventory

```diff
@@ -24,6 +24,9 @@ openshift_master_admission_plugin_config={"ValidatingAdmissionWebhook":{"configu
 # Docker
 container_runtime_docker_storage_type=overlay2
 
+# OC 3.10 node and master labeling
+openshift_node_groups=[{'name': 'node-config-infra-compute', 'labels': ['node-role.kubernetes.io/infra=true', 'node-role.kubernetes.io/compute=true']}, {'name': 'node-config-master', 'labels': ['node-role.kubernetes.io/master=true']}]
```

Comment:
setting `openshift_node_groups` will set labels automatically according to the group, so you can remove it and you must have different groups, for `master`, `infra` and `compute` resources(in general every node can server as `infra` resource, but if we talk about non-HA deployment, the cluster must have only one master node)